tests/core/mempool/test_mempool_manager.py

```diff
@@ -339,61 +339,21 @@ async def test_double_spend_prevalidation() -> None:
-    "amount,expected_status,expected_error",
-    [
-        (TEST_COIN_AMOUNT, MempoolInclusionStatus.SUCCESS, None),
-        (TEST_COIN_AMOUNT + 1, MempoolInclusionStatus.FAILED, Err.MINTING_COIN),
-    ],
-)
-async def test_minting_coin(
-    amount: uint64,
-    expected_status: MempoolInclusionStatus,
-    expected_error: Optional[Err],
-) -> None:
-    async def get_coin_record(coin_id: bytes32) -> Optional[CoinRecord]:
-        test_coin_records = {TEST_COIN_ID: TEST_COIN_RECORD}
-        return test_coin_records.get(coin_id)
-
-    mempool_manager = await instantiate_mempool_manager(get_coin_record)
-    conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, amount]]
-    try:
-        result = await generate_and_add_spendbundle(mempool_manager, conditions)
-        _, status, error = result[2]
-        assert status == expected_status
-        assert error == expected_error
-    except ValidationError as e:
-        assert expected_status == MempoolInclusionStatus.FAILED
-        assert e.code == expected_error
+async def test_minting_coin() -> None:
+    mempool_manager = await instantiate_mempool_manager(zero_calls_get_coin_record)
+    conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, TEST_COIN_AMOUNT + 1]]
+    sb = spend_bundle_from_conditions(conditions)
+    with pytest.raises(ValidationError, match="MINTING_COIN"):
+        await mempool_manager.pre_validate_spendbundle(sb, None, sb.name())
 
 
 @pytest.mark.asyncio
-@pytest.mark.parametrize(
-    "amount,expected_status,expected_error",
-    [
-        (TEST_COIN_AMOUNT, MempoolInclusionStatus.SUCCESS, None),
-        (TEST_COIN_AMOUNT + 1, MempoolInclusionStatus.FAILED, Err.RESERVE_FEE_CONDITION_FAILED),
-    ],
-)
-async def test_reserve_fee_condition(
-    amount: uint64,
-    expected_status: MempoolInclusionStatus,
-    expected_error: Optional[Err],
-) -> None:
-    async def get_coin_record(coin_id: bytes32) -> Optional[CoinRecord]:
-        test_coin_records = {TEST_COIN_ID: TEST_COIN_RECORD}
-        return test_coin_records.get(coin_id)
-
-    mempool_manager = await instantiate_mempool_manager(get_coin_record)
-    conditions = [[ConditionOpcode.RESERVE_FEE, amount]]
-    try:
-        result = await generate_and_add_spendbundle(mempool_manager, conditions)
-        _, status, error = result[2]
-        assert status == expected_status
-        assert error == expected_error
-    except ValidationError as e:
-        assert expected_status == MempoolInclusionStatus.FAILED
-        assert e.code == expected_error
+async def test_reserve_fee_condition() -> None:
+    mempool_manager = await instantiate_mempool_manager(zero_calls_get_coin_record)
+    conditions = [[ConditionOpcode.RESERVE_FEE, TEST_COIN_AMOUNT + 1]]
+    sb = spend_bundle_from_conditions(conditions)
+    with pytest.raises(ValidationError, match="RESERVE_FEE_CONDITION_FAILED"):
+        await mempool_manager.pre_validate_spendbundle(sb, None, sb.name())
```

Comment:
same here, I think we should have a positive test

Reply:
Done.